### ISSUE TITLE
docs(web): text layout style for the description about "/anonmous" extra

### ISSUE THREAD

**Issue by masatake** [universal-ctags/ctags#2826]
See the description of "/anonymous" extra in https://docs.ctags.io/en/latest/man/ctags.1.html#tags-file-contents-options .
The style of text layout is very different from that of other fields.
It is better that the styles are consistent among fields.

@hirooih, could you look into this?

I guess the following change may fix this issue. However, I have not tried my chanage on readthedocs.org.

```
diff --git a/docs/man/ctags.1.rst b/docs/man/ctags.1.rst
index 5875b0c0..dda39fe3 100644
--- a/docs/man/ctags.1.rst
+++ b/docs/man/ctags.1.rst
@@ -571,7 +571,7 @@ Tags File Contents Options
 
        The meaning of major extras is as follows (one-letter flag/long-name flag):
 
-       /anonymous
+       \/anonymous
                Include an entry for the language object that has no name like lambda
                function. This extra has no one-letter flag and is enabled by
                default. The extra tag is useful as a placeholder to fill scope fields
diff --git a/man/ctags.1.rst.in b/man/ctags.1.rst.in
index 87a55e7f..3d8e1351 100644
--- a/man/ctags.1.rst.in
+++ b/man/ctags.1.rst.in
@@ -571,7 +571,7 @@ Tags File Contents Options
 
        The meaning of major extras is as follows (one-letter flag/long-name flag):
 
-       /anonymous
+       \/anonymous
                Include an entry for the language object that has no name like lambda
                function. This extra has no one-letter flag and is enabled by
                default. The extra tag is useful as a placeholder to fill scope fields
```

**Comment by hirooih** [comment]
I already fixed this issue on my local repository.
I'll send PR with other fixes.

**Comment by masatake** [comment]
I see. Thank you.

**Comment by hirooih** [comment]
Fixed in #2842.

@masatake san
Shall I close this?

**Comment by masatake** [comment]
Sure.
